src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+public class Ui {
+    public static final String DIVIDER = "____________________________________________________________";
+    public Ui() {
+
+    }
+
+    public void showWelcome() {
```

Comment:
To standardise the method names in this class, considering changing this to showWelcomeMessage()?

Reply:
Thanks for spotting that! Will update this